src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/support/DeadlineSupportControllerTest.java

```diff
@@ -28,8 +29,11 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoInteractions;
 import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.claimstore.utils.VerificationModeUtils.once;
 
 @RunWith(MockitoJUnitRunner.class)
 public class DeadlineSupportControllerTest {
```

Comment:
Fancy migrating to JUnit 5?

Reply:
Not at the moment. Am under a deadline to get JDDO out.